apps/api/src/app/events/usecases/send-message/digest/digest.usecase.ts

```diff
@@ -61,7 +61,7 @@ export class Digest extends SendMessageType {
   private async getEvents(command: SendMessageCommand) {
     const currentJob = await this.jobRepository.findOne({ _environmentId: command.environmentId, _id: command.jobId });
 
-    if (currentJob.digest?.type === DigestTypeEnum.BACKOFF) {
+    if (currentJob && currentJob.digest?.type === DigestTypeEnum.BACKOFF) {
```

Comment:
```suggestion
    if (currentJob?.digest?.type === DigestTypeEnum.BACKOFF) {
```
In case it was warning for potentially undefined.